package.json

```diff
@@ -59,7 +59,7 @@
     "@aragon/os": "^4.2.0",
     "@aragon/apps-shared-minime": "^1.0.0",
     "eth-ens-namehash": "^2.0.8",
-    "ganache-cli": "~6.2.0",
+    "ganache-cli": "^6.7.0",
```

Comment:
@0xGabi I remember us rolling back to 6.2.0 due to a forwarding bug; did we test that it was resolved on newer ganache-core versions?

Reply:
Yes 🎉